src/components/DeleteTodosButton.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function DeleteTodosButton(props) {
+  return (
+    <button onClick={props.deleteTodos}>{props.children}</button>
```

Comment:
W tej sytuacji ten `props.deleteTodos` zostanie wywołany z obiektem Event clicknięcia. A to nie jest za dobre, bo komponen nadrzędny nie jest zainteresowany tym obiektem. 

z tego co widzę, to ten event nie jest potrzebny, żadna metoda z niego nic nie czyta. Ale musiałem przeanalizować komponenty używające `DeleteTodosButton`, żeby się o tym przekonać. 

Jakbyś to napisałą tak:

```
<button onClick={() => props.deleteTodos()}>{props.children}</button>
```

To byłby wyraźny sygnał, że obiekt eventu nie jest istotny,